tests/src/python/test_qgsproviderconnection_base.py

```diff
@@ -544,6 +545,11 @@ def testCreateSqlVectorLayer(self):
             print(f"FIXME: {self.providerKey} data provider test case does not define self.sqlVectorLayerTable for query layers test!")
             return
 
+        crs = getattr(self, 'sqlVectorLayerCrs', None)
+        if crs is None:
+            print(f"FIXME: {self.providerKey} data provider test case does not define self.sqlVectorLayerCrs for query layers test!")
+            return
```

Comment:
This should be an assert -- otherwise we risk tests which should be failing but which pass the ci anyway

Reply:
This is a generic test and not all inherited tests (hana for instance) define the schema/table/crs variables. The test is run only if the variables are defined.